test/e2e/e2e.go

```diff
@@ -23,5 +23,5 @@ import (
 func main() {
 	flag.Parse()
 	e2e.RunBasicCreateTest()
-	e2e.RunHCFromProbeTest()
+	//	e2e.RunHCFromProbeTest()
```

Comment:
Why? Remove if not required or add a comment explaining why this is commented out and probably a TODO to uncomment it when it can be uncommented.

Reply:
Good catch, uncommented